examples/apply-setters/apply-setters-simple/README.md

```diff
@@ -8,6 +8,14 @@ with setter references.
 In this example, we will see how to apply desired setter values to the 
 resource fields parameterized by `kpt-set` comments.
 
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/apply-setters/apply-setters-simple
```

Comment:
Can you do an audit of kpt.dev content that refer to example packages in the catalog? Also at some point, need to release these packages and uprev the kpt.dev content to use them.

Reply:
Will create a PR in kpt repo soon.

Reply:
https://github.com/GoogleContainerTools/kpt/pull/2247